cfgov/jinja2/v1/_includes/macros/video-player.html

```diff
@@ -36,16 +36,18 @@
     %}
     {% set button_pos   = options.button_pos or 'center' %}
     {% set is_flexible  = ( options.video.width or options.video.height ) is not defined %}
-    <div class="video-player video-player__{{ video_player }}"
+    <div class="video-player
+                video-player__{{ video_player }}
+                {{'featured-content-module_visual' if options.video.is_fcm else '' }}"
          data-video-id="{{ video_id }}"
-         data-width="{{ options.video.width }}"
-         data-height="{{ options.video.height }}">
+         data-width="{{ options.video.width if options.video.width else '' }}"
```

Comment:
What happens if nothing is passed and these are both empty? Should we use default width and height?